classes/class-wc-connect-shipping-label.php

```diff
@@ -202,9 +175,18 @@ public function get_selected_packages( WC_Order $order ) {
 							$formatted = WC_Connect_Compatibility::instance()->get_formatted_variation( $product, true );
 							$product_data['attributes'] = $formatted;
 						}
+						$customs_info = get_post_meta( $product_data['product_id'], 'customs_info', true );
+						if ( $customs_info ) {
+							$product_data = array_merge( $product_data, $customs_info );
+						}
 					} else {
 						$product_data['name'] = WC_Connect_Compatibility::instance()->get_product_name_from_order( $product_data['product_id'], $order );
 					}
+					$product_data['value'] = WC_Connect_Compatibility::instance()->get_product_price_from_order( $product_data['product_id'], $order );
+					if ( ! $product_data['value'] ) {
```

Comment:
Should this be a `null` check? Also.. if the `null` case isn't used by the callers, why not default to `0`?

Reply:
I've changed the check to be more explicit in https://github.com/Automattic/woocommerce-services/pull/1382/commits/02adfbffebef69866a3d68a2b21b4d1258e99d6e
I still think defaulting to `null` makes sense for the `get_product_price_from_order` function, since `0` is a valid price. The fact that in this case, if the price isn't founf it defaults to `0`, it should be part of the business logic, not the "compatibility" layer.